schema-next.yaml

```diff
@@ -16,10 +16,13 @@ versions:
               vcs.repository.ref.name: vcs.ref.head.name
               vcs.repository.ref.revision: vcs.ref.head.revision
               vcs.repository.ref.type: vcs.ref.head.type
-        # https://github.com/open-telemetry/semantic-conventions/pull/1440
+        # https://github.com/open-telemetry/semantic-conventions/pull/TODO
         - rename_attributes:
             attribute_map:
-              feature_flag.provider_name: feature_flag.system
```

Comment:
I'm going to extract it to another PR that we can merge right now to reduce churn in feature-flag area